spec/Knp/DictionaryBundle/Dictionary/Factory/CallableFactorySpec.php

```diff
@@ -60,3 +81,15 @@ public function getYolo()
     {
     }
 }
+
+class CallableService
+{
+    public function __invoke()
+    {
+        return [
+            'foo1' => 'bar1',
+            'foo2' => 'bar2',
+            'foo3' => 'bar3',
+        ];
+    }
+}
```

Comment:
Last little change, can you add a new line at end of file? :)

Reply:
It's done ;)